DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
-Package: admiraltemplate
-Title: ADaM in R Asset Library - Extension
-Version: 0.0.3
+Package: `<<INSERT PACKAGE EXT NAME>>'
```

Comment:
This crashes our checks - maybe we could have a startup message that refers to a function that when calls updates everything for the extension pkg?  Maybe fun issue for someone to work on from community?

Reply:
nice idea for the function (although in reality its questionable whether worth the investment given it'd only ever be needed once or twice a year likely). for the CI i think fine for them to crash for admiraltemplate as its not a real package anyway

Reply:
Should we mention that once you fix this the ci will work again?

Reply:
not sure where would you put this note. it feels obvous for me as there are lots of places you need to update the placeholders, and the person first making an admiral pkg ext is never going to be a complete newbie. feel free to make a commit though if you think thats useful.

Reply:
I put it to the community to update if they want to - but agree low priority, but a fun one. https://github.com/pharmaverse/admiraltemplate/issues/80